xmpsdk/include/client-glue/TXMPMeta.incl_cpp

```diff
@@ -90,7 +90,7 @@ Terminate()
 }
 
 // =================================================================================================
-// Constuctors, destructor, operators
+// Constructors, destructor, operators
 // ==================================
```

Comment:
Here and below are places where apparently the bottom `=`/`-` should be as long as the text...